R/aggregate.R

```diff
@@ -81,12 +86,6 @@ agg_climate_data <- function(year, data_raster, climate_var, daily_agg, trans =
   ## Average
   if(daily_agg == 'average'){
 
-    # Check if there are 24*365 or 24*366 layers
-    if(!raster::nlayers(clim_raster) %in% c(8760, 8784)){
-
-      message(crayon::red("Warning: Incomplete year of data; raster has", length(all_layers),
```

Comment:
did this kind of warning get reused elsewhere? i think it's critical to confirm the right layers are there (and in the right dimensions).

Reply:
The idea is that the package takes any number of days in case they haven't downloaded their data into single year raster bricks. This does bring up some other edge case issues and is worth discussing.

Reply:
This is the only place that checks for the number of layers in the original raster brick, a similar warning doesn't show up at other points in the pipeline (as far as I know). But I think Tyler's idea to use `lubridate` to extract dates from the layer names (related to another comment) might prevent other issues that would arise if the assumed structure (24 layers per day) isn't met. This structure doesn't rely on there being 365 days but it does rely on there being 24 layers per day and it assumes the first 24 layers are one day, the next 24 layers are a second day etc.

Reply:
We'll have to discuss how to handle weird cases like someone using data that starts at a time other than midnight, but I think right now as long as the number of layers is a multiple of 24 and beings at midnight, this should aggregate to the daily level. Being able to take inputs that are not hourly is a slightly different and more complex problem, one that this warning doesn't exactly solve. We could check to make sure the number of layers is divisible by 24 though